apps/server/src/modules/pseudonym/service/pseudonym.service.ts

```diff
@@ -81,6 +86,7 @@ export class PseudonymService {
 			this.deletePseudonymsByUserId(userId),
 			this.deleteExternalToolPseudonymsByUserId(userId),
 		]);
+		this.logger.log({ action: 'Deleted Pseudonyms for user ', userId });
```

Comment:
Please log some more metadata (as for the _classes_ cases where I've left some example).